### ISSUE TITLE
New forge release

### ISSUE THREAD

**Issue by baurmatt** [electrical/puppet-lib-file_concat#39]
Hey, could you please release a new version to the forge?

**Comment by electrical** [comment]
Hi @baurmatt this library was picked up and improved by the puppetlabs concat module.
I would advise to use that module.

**Comment by baurmatt** [comment]
Yeah i know -.- We just can't upgrade our dependencies that easily and i hoped you might have an easy way to release a new version to the forge :)

**Comment by electrical** [comment]
Ahh okay. I see. I haven't been keeping track of the puppetlabs concat changes at all so i'm afraid it won't be that easy.
